action.yaml

```diff
@@ -19,6 +19,21 @@ action:
       description: Allow override committed version by current build value
       type: boolean
       default: false
+    - name: playbook-filter
+      title: Filter resources by playbook usage
+      description: Parse playbooks and propagate used resources only
+      type: boolean
+      default: true
+    - name: hide-progress
+      title: Hide progress
+      description: Don't draw progress bars (true if log level above 0)
+      type: boolean
+      default: false
+    - name: time-depth
+      title: Time depth
+      description: Use commits only after specific date (ex. 2006-01-02)
```

Comment:
`2006-01-02` let's change for something more obvious like `2006-12-30` or `yyyy-mm-dd`